argh/src/lib.rs

```diff
@@ -799,6 +799,15 @@ impl Flag for bool {
     }
 }
 
+impl Flag for Option<bool> {
+    fn default() -> Self {
+        None
+    }
```

Comment:
I'd expect there to be a blank line between these. Did this run through `cargo fmt`?

Reply:
`cargo fmt` was happy without it, but this indeed would look cleaner.